engine/gfx/sprite_anims.asm

```diff
@@ -220,7 +220,7 @@ DoAnimFrame:
 	callfar ComposeMail_AnimateCursor
 	ret
 
-.GameFreakLogo:
+.GameFreakLogo
```

Comment:
Can we consistently have colons for the labels that correspond to `SPRITE_ANIM_SEQ_*` constants? Right now some do, some don't.

Reply:
I just looked at the surrounding stuff and nothing had it, so removed it. Will just add to everything.

Reply:
This will be left for a different PR, where global labels will be created, since there's many locals in this file.